src/@next/components/molecules/TextField/TextField.tsx

```diff
@@ -8,13 +8,34 @@ import { ErrorMessage, Input } from "@components/atoms";
 export const TextField: React.FC<IProps> = ({
   errors,
   helpText,
+  required,
+  onBlur,
+  value,
   ...rest
 }: IProps) => {
+  const [touched, setTouched] = useState(false);
```

Comment:
touched state should be kept in the parent form state container (Formik does that out of the box)

Reply:
This will be fixed in different PR as this needs some bigger logic changes for handling formik and backend errors